+nosnoc/+ocp/Solver.m

```diff
@@ -90,11 +90,18 @@
             end
         end
 
-        function solve(obj, plugin)
+        function solve(obj)
             arguments
                 obj
-                plugin = 'reg_homotopy'
             end
+            switch class(obj.solver_opts)
+              case "nosnoc.reg_homotopy.Options"
+                plugin = 'reg_homotopy';
+                obj.solver_opts.assume_lower_bounds = true;
```

Comment:
okay. now I see, the answer to my previous comment. would you be so kind to add here the comment from discrete_time_problem I added in #145